src/Client/ConcordiumClient.cs

```diff
@@ -94,14 +94,14 @@ SignedAccountTransaction<T> transaction
     /// case, then the sequence number is reliable.
     /// </returns>
     /// <exception cref="RpcException">The call failed.</exception>
-    public (Types.AccountSequenceNumber, bool) GetNextAccountSequenceNumber(
-        Types.AccountAddress accountAddress
+    public (AccountSequenceNumber, bool) GetNextAccountSequenceNumber(
+        AccountAddress accountAddress
```

Comment:
Generally don't do this, i.e., make many unrelated changes in many places.

PRs should be as minimal as reasonable, and we should not be making multiple independent changes everywhere.

Reply:
I was told to run `dotnet format` such that code aligned with our editerconfig settings - this is why these changes are present.